testdata/repros/Issue2984.t.sol

```diff
@@ -12,8 +12,8 @@ contract Issue2984Test is DSTest {
 
     function setUp() public {
         fork = vm.createSelectFork(
-            "https://api.avax-test.network/ext/bc/C/rpc",
-            12880747
+            "rpcAlias",
+            7627763
```

Comment:
Does this work with `rpcAlias`?

Reply:
Yes, test `test_issue_2984` which runs this sol test pass